src/fragments/lib/datastore/ios/data-access/query-predicate-multiple-snippet.mdx

```diff
@@ -7,7 +7,7 @@ let p = Post.keys
 do {
     let result = try await Amplify.DataStore.query(
         Post.self,
-        where: p.rating > 4 && p.status == PostStatus.published
+        where: p.rating > 4 && p.status == PostStatus.active
```

Comment:
In V2, shouldn't this be .published?

Reply:
i searched for all places for "enum PostStatus" to find the schemas and it appears they are all containing ACTIVE and INACTIVE, can't find which schema generates Published